pkg/kubelet/hosts_test_file1

```diff
@@ -0,0 +1,8 @@
+# hosts file for testing.
```

Comment:
Better to create the test file dynamically when running tests, e.. under `/tmp` dir. The test could be useful when hostAliases is introduced.